src/main/java/org/apache/commons/collections4/MapBuilder.java

```diff
@@ -0,0 +1,115 @@
+package org.apache.commons.collections4;
+
+import org.apache.commons.collections4.map.HashedMap;
+
+import java.util.*;
+
+/**
+ * Defines an Helper Builder that generates a {@code Map}
+ * A Builder class to help decide which type of map to use based on simple requirements.
+ * Currently It takes care of only basic types of Maps and can be extended to different types of Maps available in the ecosystem.
+ *
+ * <pre>{@code
+ * Map builderMap = new MapBuilder().setIterationOrder(MapBuilder.KeyOrder.INSERTION_ORDER).build();
+ * builderMap.put("A", 1);
+ * builderMap.put("X", 24);
+ * builderMap.put("B", 2);
+ * builderMap.put("Y", 26);
+ * }</pre>
+ *
+ * @author Amita Pradhan
```

Comment:
We have dropped author tags (it will be recorded in git, and also in our changelog)

Reply:
Done. Removed.